README.md

```diff
@@ -31,12 +31,59 @@ cd dianna
 python3 -m pip install -e .
 ```
 
+
 ## Documentation [![Documentation Status](https://readthedocs.org/projects/dianna/badge/?version=latest)](https://dianna.readthedocs.io/en/latest/?badge=latest)
 
+### Benchmark datasets
+(TODO: add a sentence and/or image for each and the Zenodo and/or original links and links to notebooks for creating them)
+
+DIANNA offers to use simple benchamrk datasets for evaluating and comparing the XAI methods:
+
+#### Images
+* Binary (2-class) MNIST
+* Simple Geometric (triangles and circles)
+* Simple Scientific (LeafSnap30)
+
+#### Text
+* Movie reviews treebank
+
+### ONNX models
+(TODO: Add all links) 
+
+**We work with ONNX!** ONNX is a great unified oen NN standard which can be used to boost reproducible science. Your ONXX model also gives you boost in performance! In case your models are still in another popular DNN format, here are the simple recipies on to convert them:
+* pytorch
+* tensorflow
+* keras
+* scikit-learn
+
+And here are how we created our models on the benchmark datasets:
+
+#### Images
+* Binary MNIST model
+* Simple Geometric model
+* Simple Scientific model
+
+#### Text
+* Movie reviews model
+
+**_We envision the birth of the ONNX Scientific models zoo soon..._**
+
+### Tutorials
+DIANNA supports the following data modalities and XAI methods:
+(TODO: add links to tutorials in table?)
+
+|Data \ XAI|RISE|LIME|KernelSHAP|
+|-----|---|---|---|
+|Images|V|V|V|
+|Text|V|V|planned|
```

Comment:
```suggestion
|Images|:white_check_mark:|:white_check_mark:|:white_check_mark:|
|Text|:white_check_mark:|:white_check_mark:|planned|
```

Apart from that, the distinction between planned and coming soon is a bit weird, maybe just one category of "later" will do.

Reply:
Shall we address it also in a separate Roadmap.md?